src/LaravelDataTables/Providers/ServiceProvider.php

```diff
@@ -1,11 +1,17 @@
 <?php namespace SevenD\LaravelDataTables\Providers;
 
+use App\Mail\DataTablesCsvEmail;
```

Comment:
Shouldn't import a class from "App" namespace as this is definitively referencing the cloud-connect repo.

Instead use the IoC Container and a Contract.